components/script/dom/htmlinputelement.rs

```diff
@@ -1034,8 +1034,15 @@ impl HTMLInputElement {
             InputType::Time => {
                 let mut textinput = self.textinput.borrow_mut();
 
-                if ! textinput.single_line_content().is_valid_time_string() {
-                    *textinput.single_line_content_mut() = "".into();
+                if !textinput.single_line_content().is_valid_time_string() {
+                    textinput.single_line_content_mut().clear();
+                }
+            }
+            InputType::DatetimeLocal => {
+                let mut textinput = self.textinput.borrow_mut();
+                if textinput.single_line_content_mut()
+                    .convert_valid_normalized_local_date_and_time_string().is_err() {
+                        textinput.single_line_content_mut().clear();
```

Comment:
Nit:

Can it be indented like following?

```rust
    if textinput.single_line_content_mut()
                      .convert_valid_normalized_local_date_and_time_string()
                      .is_err() {
```

Reply:
sure. it's ok. :)

Reply:
@CYBAI I'm actually not sure what the proper rustfmt for this is, I suspect it is this instead?
```rust
if textinput.single_line_content_mut()
    .convert_valid_normalized_local_date_and_time_string()
    .is_err()
{
    // body goes here
}
```
but since we don't run rustfmt yet, I don't really care too much as long as ./mach test-tidy says it's ok.

Reply:
@KiChjang Yes, I think the left curly brace will be next line.

Ref: https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/expressions.md#line-breaking